packages/desktop-client/src/components/mobile/budget/index.tsx

```diff
@@ -343,9 +343,11 @@ function BudgetInner(props: BudgetInnerProps) {
 
   const onEditCategory = id => {
     const category = categories.find(c => c.id === id);
+    const categoryGroup = categoryGroups.find(g => g.id === category.cat_group);
     dispatch(
       pushModal('category-menu', {
         categoryId: category.id,
+        isHidden: categoryGroup?.hidden,
```

Comment:
Nit: This should be named as `isGroupHidden` or just pass in the `categoryGroup` similar to the other one :)